osu-ac/cacher.py

```diff
@@ -11,7 +12,7 @@ class Cacher:
     methods provide cleaner access than passing around a Cacher class.
     """
 
-    conn = sqlite3.connect(PATH_DB)
+    conn = sqlite3.connect(str(PATH_DB))
```

Comment:
sqlite3.connet takes in a path like object (see https://docs.python.org/3.7/library/sqlite3.html#sqlite3.connect), which PATH_DB is. No need to cast to str

Reply:
I take this back, python3.6 sqlite3 does not support path0-like objects (https://docs.python.org/3.6/library/sqlite3.html#sqlite3.connect), which we definitely want to support.